### DISCUSSION TITLE
Can i use helmet.js with next.js?

### DISCUSSION THREAD

**Discussion by minminminmmm** [vercel/next.js#27108] [Help] [answered]
I always use express to develop back end. but our team decided to use next.js.
so, i heard that next.js own server is made of express?
Can i use [helmet library](https://helmetjs.github.io/) with next.js?
or Please tell me other security tools that can be used with next.js

**Comment by SidKH** [comment, marked as answer]
Have you seen this?
https://github.com/serverless-nextjs/serverless-next.js/issues/433#issuecomment-779712861
